setup/service/dhis2-create-instance

```diff
@@ -159,34 +159,51 @@ sudo sed -i "/$IP/d" /etc/hosts
 sudo sed -i "$ a\
   $IP $NAME" /etc/hosts
 
-hosts="$host";
-remote_addr="$remote_addr";
-proxy_add_x_forwarded_for="$proxy_add_x_forwarded_for";
+
+PROXY=$(cat configs/containers.json| jq -r .proxy)
```

Comment:
We need a smarter way to discover proxy type.  This only works when run from setup directory I think